### DISCUSSION TITLE
Can I use a dynamically created blob URL with Scene Viewer?

### DISCUSSION THREAD

**Discussion by jakecjones** [google/model-viewer#1863] [Q&A]
Model viewer works great with my blob to create a dynamic GLB model but scene viewer only works with my static GLB model and won't accept my blob. Is there a workaround or something the model viewer team has already addressed?

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions
- model-viewer: v0.3.1
- three.js: r0.105.2
![Screenshot_20190607-212017](https://user-images.githubusercontent.com/31197056/59141935-6e34bf00-896a-11e9-87c0-7cb7652a92a2.png)
![Screenshot_20190607-212026](https://user-images.githubusercontent.com/31197056/59141936-6e34bf00-896a-11e9-8e97-6c3cc8cbd204.png)
![Screenshot_20190607-212042](https://user-images.githubusercontent.com/31197056/59141937-6ecd5580-896a-11e9-80ca-e20c602dc5c8.png)

**Comment by cdata** [comment]
Unfortunately, there is no workaround for this at this time. Scene Viewer is a separate app entirely from your web browser, and there is no way to pass the blob out of the browser and into Scene Viewer.
